### PR TITLE
Major refactor to simplify webkms server implementations.

### PR DESCRIPTION
**Pull request by dlongley**

### CODE REVIEW

File: lib/schemas/GenerateKeyOperation.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
 /*!
  * Copyright (c) 2019-2021 Digital Bazaar, Inc. All rights reserved.
  */
-'use strict';
-
-const context = require('./generateKeyOpContext.js');
+import {default as context} from './generateKeyOpContext.js';
 
 module.exports = {
```
Comment (davidlehn) [inline lib/schemas/GenerateKeyOperation.js:6]: Switch to esm.

File: package.json [hunk 2]
```diff
@@ -29,14 +29,18 @@
     "lib": "./lib"
   },
   "dependencies": {
+    "@digitalbazaar/ezcap-express": "^3.2.0",
+    "@digitalbazaar/http-digest-header": "^1.0.0",
     "ajv": "^6.0.0",
     "base58-universal": "^1.0.0",
+    "express-async-handler": "^1.1.4",
```
Comment (davidlehn) [inline package.json:36]: Missing `esm` and other related field updtes for index vs main.

Comment (dlongley) [reply package.json:37]: What are the "other related field updates"?

Comment (davidlehn) [reply package.json:37]: The ones we haven't figured out yet.  Whatever combination of "main", "module", "exports", etc are needed to make cjs + esm work properly.  Looks like just "main" is there now and would load `lib/index.js`, but nothing points at `lib/main.js` for ESM use.

Comment (dlongley) [reply package.json:37]: Add `module` for now to address the main point. Not dealing with the larger/systemic issue at this time.

**Comment by dlongley** [comment]
Ready for wider review, but not ready for release yet.

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

**Pull request merged**